app/controllers/admin_public_body_controller.rb

```diff
@@ -142,10 +142,12 @@ def mass_tag_add
   def missing_scheme
     # There might be a way to do this in ActiveRecord, but I can't find it
     @public_bodies = PublicBody.find_by_sql("
-            SELECT a.id, a.name, a.url_name, COUNT(*) AS howmany
```

Comment:
So:

- no specs fail when the suite is run before this change is made
- adding a simple `expect(response).to be_success` spec illustrates a failure exists
- there's a problem with a different query in this action
- you believe there to be a problem with the `find_by_sql` query, but there aren't any specific specs to demonstrate its behaviour?

Is my understanding of this correct?

If so, I think plan of action is:

- Push the `find_by_sql` in to `PublicBody.missing_scheme` and add some tests
- Add specs for `AdminPublicBodyController#missing_scheme`

…or we could just leave it broken / delete it because it's not linked from anywhere so presumably (we could answer this if we had https://github.com/mysociety/alaveteli/issues/2945) isn't used.